### DISCUSSION TITLE
Replicating Files from Minio to AWS S3: Seeking Insights on Best Approach

### DISCUSSION THREAD

**Discussion by OkD0Cs** [minio/minio#17316] [Q&A] [answered]
Hello everyone,

I'm currently running Minio on a Docker container in a Synology NAS, and it's functioning properly. My objective is to replicate certain folders from Minio to an AWS S3 bucket. This replication should be one-way, meaning that any new item created in Minio should also be created in the specified bucket.

To achieve this, I have created two aliases using the `mc alias` command and started the replication between them with `mc mirror`. However, I'm unsure if this is the best approach, as the command is being run from an AWS EC2 instance that was specifically set up for this purpose. I would appreciate any insights or suggestions on how to handle this type of situation effectively.

Thank you in advance for your assistance.

**Comment by klauspost** [comment, marked as answer]
Unless you are running MinIO on both ends, `mc mirror` is your best option.
